docs/examples/drag-and-drop/script.js

```diff
@@ -0,0 +1,85 @@
+function $(query) {
+    return document.querySelector(query)
+}
+
+const uploadBox = $('#uploadBox')
+uploadBox.ondragover = () => false
+uploadBox.ondrop = async (e) => {
+    e.preventDefault()
+    const { dataTransfer } = e
+    if(!dataTransfer) return
+
+    for(const file of dataTransfer.files) {
+        await uploadFile(file)
+    }
+}
+
+const uploadListBox = $('#uploadListBox')
+
+const protocolSelect = $('#protocolSelect')
+
+async function uploadFile(file) {
+    const name = file.name;
+    const buffer = await file.arrayBuffer();
```

Comment:
Instead of getting the buffer out, we should upload using the `FormData` support for the fetch body. That should be a lot faster

Reply:
I think this would also allow for uploading several files at once which could be useful if somebody wants to make a site from several files

Reply:
Now Uploading with FormData. It is indeed much faster and it does allow for site upload.